sklearn/metrics/_ranking.py

```diff
@@ -1685,10 +1694,12 @@ def ndcg_score(y_true, y_score, *, k=None, sample_weight=None, ignore_ties=False
     if y_true.min() < 0:
         # TODO(1.4): Replace warning w/ ValueError
         warnings.warn(
-            "ndcg_score should not be used on negative y_true values. ndcg_score will"
-            " raise a ValueError on negative y_true values starting from version 1.4.",
+            "ndcg_score should not be used on negative y_true values. ndcg_score"
+            " will raise a ValueError on negative y_true values starting from"
+            " version 1.4.",
```

Comment:
It would be best to not add changes that are not related to the PR.

Reply:
This change results from running `black` in the `pre-commit` hook. Reverting it seems impossible to me without disabling the hook. What would be the preferred way of working here? Please advise.

Reply:
If `black` is the culprit so we can let it then. I am not really sure why this is triggered now and not when we last apply it to format all files.

Reply:
maybe a different version of black. It should be pinned to 22.3.0 (maybe time to upgrade though)